src/gnn_reco/models/convnet.py

```diff
@@ -0,0 +1,125 @@
+import torch
```

Comment:
As mentioned in the general comments: As the focus is on integrating model training code in the repo quickly, I would review the model-specific files as they already work.

Reply:
OK.